hackathon/templates/hackathon/final_score.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<h1 class="table_titles"> {{ hackathon }} <br> Final Scores </h1>
+	{% if sorted_teams_scores %}
+	<table class="final_score_table">
+		<tr class="table_titles">
```

Comment:
Would it be more semantically acceptable to use `<thead>` and then `<tbody>` for this particular table?